src/components/forms/dashboard/links/csv-shorten.tsx

```diff
@@ -0,0 +1,93 @@
+"use client";
+
+import Papa from "papaparse";
+import { toast } from "sonner";
+
+import { quickLinkShorten } from "@/actions/link-actions";
+import { Button } from "@/components/ui/button";
+
+import { useTransition } from "react";
+
+const CsvShortenForm = () => {
+  const [loading, startTransition] = useTransition();
+  interface CSVRow {
+    links: string;
+  }
```

Comment:
We should move the interface outside of the function so it's definition comes before the `CsvShortenForm`